pkg/asset/installconfig/aws/permissions.go

```diff
@@ -193,14 +192,14 @@ func ValidateCreds(ssn *session.Session) error {
 		return errors.Wrap(err, "getting creds from session")
 	}
 
-	client, err := ccaws.NewClient([]byte(creds.AccessKeyID), []byte(creds.SecretAccessKey))
+	client, err := ccaws.NewClient([]byte(creds.AccessKeyID), []byte(creds.SecretAccessKey), "openshift-installer")
```

Comment:
I would like to pass something like https://github.com/openshift/installer/blob/e1e3c787398632a227df9b0598095536cf2e3547/pkg/asset/installconfig/aws/aws.go#L108 to the client..